guess-the-number/guess_the_number/guess.py

```diff
@@ -8,6 +8,13 @@
 # Read https://h2oai.github.io/wave/docs/tutorial-counter#step-1-start-listening
 from h2o_wave import Q, app, main, ui
 
+zones = [
+    ui.zone(name='title', size='60px'),
+    ui.zone(name='banner', direction=ui.ZoneDirection.ROW,
+            wrap='start', justify='center'),
+    ui.zone(name='content', direction=ui.ZoneDirection.ROW,
+            wrap='start', justify='center'),
+]
```

Comment:
The goal of this issue is to only add theme toggle, the layout should be left untouched, please revert.